ramips: use led, function/color syntax for some routers

namiltd opened the pull request:
- Rename the mt7621_dlink_dir-xx60-a1.dtsi file to a more appropriate name.
- Abandoning the label and using the function/color syntax for some dlink dir series routers: dir-1960-a1, dir-2660-a1, dir-2640-a1, dir-3040-a1 and dir-3060-a1
- Setting the LED name and abandoning the label and using the function/color syntax for some TP-Link Archer series routers: Archer C2 v1, Archer C20 v1, Archer C20i and Archer C50 v1

Lucky1openwrt commented:
ok cool I guess with v24 if you sysupgrade you have to fix the leds with the version change
I think you have to change the aliases as well tho
```

	aliases {
		label-mac-device = &gmac0;
		led-boot = &led_power_orange;
		led-failsafe = &led_power_white;
		led-running = &led_power_white;
		led-upgrade = &led_net_orange;
	};
```

namiltd commented:
Migration done.

Lucky1openwrt commented:
all good :)
gave it a test run on my DIR-1960-A1 
all worked even the Migration :)

Lucky1openwrt commented on target/linux/ramips/image/mt7621.mk:
```diff
@@ -763,35 +763,35 @@ endef
 
 define Device/dlink_dir-3060-a1
-  $(Device/dlink_dir-xx60-a1)
+  $(Device/dlink_dr_nand_128m)
```
missing an i in $(Device/dlink_dr_nand_128m)

namiltd replied:
Fixed.

namiltd committed:
ramips: rename include file to a more suitable name

Rename from mt7621_dlink_dir-xx60-a1.dtsi to mt7621_dlink_dir_nand_128m.dtsi
and associated group name when creating the mt7621.mk image

Co-authored-by: Alan Luck <luckyhome2008@gmail.com>
Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd commented:
Add in description: Successor of PR https://github.com/openwrt/openwrt/pull/15275

namiltd committed:
ramips: use led, function/color syntax for some tp-link archer routers

Setting the LED name and abandoning the label and using the
function/color syntax for some TP-Link Archer series routers:
Archer C2 v1, Archer C20 v1, Archer C20i and Archer C50 v1

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

namiltd committed:
ramips: use function/color syntax for some dlink dir routers

Abandoning the label and using the function/color syntax for some dlink
dir series routers: dir-1960-a1, dir-2660-a1, dir-2640-a1, dir-3040-a1
and dir-3060-a1

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>
